README-automember.md

```diff
@@ -122,6 +122,7 @@ Variable | Description | Required
 -------- | ----------- | --------
 `ipaadmin_principal` | The admin principal is a string and defaults to `admin` | no
 `ipaadmin_password` | The admin password is a string and is required if there is no admin ticket available on the node | no
+`ipaapi_ldap_cache` | Set IPA connecton ldap_cache. (bool) | no
```

Comment:
I think this description needs a bit more information. "Use LDAP cache for IPA connection. The bool setting defaults to `yes`."

Reply:
Fixed.